github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-periodics.yaml

```diff
@@ -97,15 +97,21 @@ periodics:
       secret:
         secretName: gcs
     containers:
-    - image: quay.io/kubevirt/builder:2105121048-a05ef0ee1
+    - image: quay.io/kubevirt/builder:2108311443-370e711b8
```

Comment:
Where does the builder image come from? I couldn't find it under `/images`. It would be very nice to add bazelisk to this image so that we can handle projects requesting different bazel versions with the same image.

Reply:
This job is running make target `bump-kubevirtci` from kubevirt/kubevirt, so it requires to use the kubevirt builder image which comes from here: https://github.com/kubevirt/kubevirt/tree/main/hack/builder

I updated the bazel version and also a python rule with this PR: https://github.com/kubevirt/kubevirt/pull/6326

Reply:
I'm not sure whether the kubevirt builder is required to run the make target. It may make sense to just use add the tools to bootstrap image. This way we might also get rid of the kubevirt/builder image in a lot of other jobs, which I find a bit odd, now that I see it.

Reply:
I've created an image with pr-creator for that, we might just be able to replace the kubevirt/builder with it: https://github.com/kubevirt/project-infra/pull/1536

Reply:
awesome, thanks!